KOGITO-5708 update release notes and add doc regarding keycloak client config (#3841)
kiegroup/kie-docs: doc-content/kogito-docs/src/main/asciidoc/bpmn/chap-kogito-developing-process-services.adoc

* You have cloned the https://github.com/kiegroup/kogito-apps/tree/master/management-console[{PRODUCT} Management Console] repository from GitHub.
* An instance of Keycloak server is running.

> IMPORTANT: Make sure to login into the administration console of keycloak, navigate to the respective client configuration and add *** or the origin urls as a parameter to the *Web Origins*. This prevents issues related to `Access-Control-Allow-Origin`. 

Reviewer: ```suggestion
IMPORTANT: Make sure to login into the administration console of keycloak, navigate to the respective client configuration and add the correct urls corresponding to the different applications (or * for easier configuration but it could cause security issues) or the origin urls as a parameter to the *Web Origins*. This prevents issues related to `Access-Control-Allow-Origin`. 
```

Author: updated , Thanks

> IMPORTANT: Make sure to login into the administration console of keycloak, navigate to the respective client configuration and add the correct urls corresponding to the different applications (or * for easier configuration but it could cause security issues) or the origin urls as a parameter to the *Web Origins*. This prevents issues related to `Access-Control-Allow-Origin`. 

Reviewer 2: ```suggestion
IMPORTANT: Ensure that you log in to the administration console of the Keycloak and navigate to the respective client configuration. For easier configuration, you can add correct URLs of the different applications, however, it can cause security issues. You can also add the origin URL as a parameter to the *Web Origins*, which can prevent the issues related to `Access-Control-Allow-Origin`.
```

* You have cloned the https://github.com/kiegroup/kogito-apps/tree/master/task-console[{PRODUCT} Task Console] repository from GitHub.
* An instance of Keycloak server is running.

> IMPORTANT: Make sure to login into the administration console of keycloak, navigate to the respective client configuration and add *** or the origin urls as a parameter to the *Web Origins*. This prevents issues related to `Access-Control-Allow-Origin`. 

Reviewer: ```suggestion
IMPORTANT: Make sure to login into the administration console of keycloak, navigate to the respective client configuration and add the correct urls corresponding to the different applications (or * for easier configuration but it could cause security issues) or the origin urls as a parameter to the *Web Origins*. This prevents issues related to `Access-Control-Allow-Origin`. 
```

Author: updated , Thanks

> IMPORTANT: Make sure to login into the administration console of keycloak, navigate to the respective client configuration and add the correct urls corresponding to the different applications (or * for easier configuration but it could cause security issues) or the origin urls as a parameter to the *Web Origins*. This prevents issues related to `Access-Control-Allow-Origin`.

Reviewer 2: ```suggestion
IMPORTANT: Ensure that you log in to the administration console of the Keycloak and navigate to the respective client configuration. For easier configuration, you can add correct URLs of the different applications, however, it can cause security issues. You can also add the origin URL as a parameter to the *Web Origins*, which can prevent the issues related to `Access-Control-Allow-Origin`.
```
